Intersect.Editor/Core/Graphics.cs

```diff
@@ -990,21 +995,41 @@ private static void DrawSelectionRect()
 
                             if (spawnTex != null)
                             {
+                                // Check if the current spawn is selected
+                                var spawnColor = System.Drawing.Color.White;
+                                var selectedNpcIndex = Globals.MapLayersWindow.lstMapNpcs.SelectedIndex;
```

Comment:
Store these big repeated calls to vars

Reply:
we had tmpMap stored at the very beggining 🤣 
![image](https://github.com/AscensionGameDev/Intersect-Engine/assets/17498701/30c576ba-b14a-4606-9438-4ffe94346aff)